packages/googlechrome.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,54 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+    . $toolsPath\sfta.ps1
+    . $toolsPath\helpers.ps1
+
+    $downloadDir = Get-PackageCacheLocation
+    $installer          = 'googlechromeinstaller.msi'
+    $packageArgs        = @{
+        packageName     = $env:ChocolateyPackageName
+        file            = Join-Path $downloadDir $installer
+        url             = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise.msi'
+        url64bit        = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise64.msi'
+        fileType        = 'MSI'
+        silentArgs      = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+        validExitCodes  = @(0)
+    }
+
+    # Download the installer
+    $packageArgs['file'] = Get-ChocolateyWebFile @packageArgs
+
+    $sigValid = (Get-AuthenticodeSignature -FilePath $packageArgs['file']).Status -eq 'Valid'
```

Comment:
I wonder if it is enough with checking that it has a valid signature of if we need to check that the signer certificate comes from Google Chrome as well? @vm-packages opinions? (We can extract this signature to a helper for other tools in the future, so it is a general question)

Reply:
This is another function that has been on my list; `VM-Assert-Signature`. I think we should maybe open a PR for this function and hash it out there

Reply:
Personally I think I'd be ok with just checking if it is Valid since that will check it against CAs, so self signed binaries wont be shown as valid

Reply:
@day1player I think we should either introduce `VM-Assert-Signature` or a helper function to install tools using the signature instead of the hash.